lib/ghtorrent/adapters/mongo_persister.rb

```diff
@@ -60,6 +60,12 @@ def store(entity, data = {})
       mongo[entity].insert_one(data).to_s
     end
 
+    def replace(entity, query, new_entry, upsert = true)
+      check_entity_exists(entity)		
```

Comment:
Please refer to above response. If we were to add a base class method `def replacement(...)` it would be functionally identical to `check_entity_exists`